jdaviz/configs/default/plugins/plot_options/plot_options.vue

```diff
@@ -19,6 +19,15 @@
       <v-row class="row-no-outside-padding">
         <jupyter-widget v-if="selected_viewer" :widget="viewer_widget"></jupyter-widget>
       </v-row>
+
+      <v-row v-if="['specviz', 'mosviz'].indexOf(config)!==-1">

```

Comment:
`_toggle_uncertainty` currently handles specviz and cubeviz, but this says specviz and mosviz.  Also see comment above about spectrum viewers in specviz2d and mosviz.

Reply:
So if I only want it to not appear in Imviz, do I place 'imviz' in the brackets and set `.indexOf(config)!==1`? Or leave it as `-1`?

Reply:
yes, -1 is returned if the item isn't in the list, so `list.indexOf(item)!==-1` is equivalent to python's `item in list`